daemon/container_operations_unix.go

```diff
@@ -360,7 +360,7 @@ func detachMounted(path string) error {
 }
 
 func isLinkable(child *container.Container) bool {
-	// A container is linkable only if it belongs to the default network
+	// A container is linkable only if it belongs to the default network or user-definded network
```

Comment:
well ... though the meaning of your comment is correct, it is not appropriate in the context of this function. That's because `isLinkable` today assumes the Legacy Link functionality and hence the only supported network is the default network. Also the following code accepts just the default network. Hence please revert this comment.